models/Document/Editable/Video.php

```diff
@@ -1036,7 +1036,7 @@ private function getEmptyCode(): string
             $height .= 'px';
         }
 
-        return '<div id="pimcore_video_' . $this->getName() . '" class="pimcore_editable_video"><div class="pimcore_editable_video_empty" id="' . $uid . '" style="width: ' . $width . '; height: ' . $height . ';"></div></div>';
+        return '<div id="pimcore_video_' . $this->getName() . '" class="pimcore_editable_video"><div class="pimcore_editable_video_empty" id="' . $uid . '" style="width: ' . $width . '; height: ' . $height . ';" data-type="pimcore_video_editable"></div></div>';
```

Comment:
Thank you for the PR. But, the ´data-type` attribute (data-type="pimcore_video_editable") is required for videos inside the iframe only.

```suggestion
        return '<div id="pimcore_video_' . $this->getName() . '" class="pimcore_editable_video"><div class="pimcore_editable_video_empty" id="' . $uid . '" style="width: ' . $width . '; height: ' . $height . ';"></div></div>';
```

Reply:
This is from #12909

Reply:
Yes, we added it only for the video inside the iframe